pandas/_libs/parsers.pyx

```diff
@@ -741,8 +740,7 @@ cdef class TextReader:
                           self.parser.lines < hr):
                     msg = self.orig_header
                     if isinstance(msg, list):
-                        msg = "[%s], len of %d," % (
-                            ','.join(str(m) for m in msg), len(msg))
+                        msg = f"[{','.join(str(m) for m in msg)}], len of {len(msg)},"
```

Comment:
what do you think about taking the .join outside of the fstring?

Reply:
I do think that this is better(taking the join out of the f-string), 
but how would you implement this?

by assigning another variable the values from the comprehension list? then just puting it in the f-string?

I'm not really confident about this at all.

Reply:
yah, just on the previous line `joined = ','.join(str(m) for m in msg)`

> I'm not really confident about this at all.

It's really helpful that you're doing this, especially in the cython code which is harder to lint and still has some ancient idioms floating around.